master/reference/calicoctl/resources/bgppeer.md

```diff
@@ -30,31 +30,31 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema |
 |-------------|-----------------------------|-------------------|--------|
-| scope    | Determines the Calico nodes to which this peer applies. | global, node | string |
-| node     | Must be specified if scope is node, and must be omitted when scope is global. | The hostname of the node to which this peer applies. | string |
-| peerIP   | The IP address of this peer. | Valid IPv4 or IPv6 address.  | string |
+| name     | Unique name to describe this resource instance. Must be specified.| Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
 
 #### Spec
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
+| node     | If specified, the scope is node level, otherwise the scope is global. | The hostname of the node to which this peer applies. | string | |
+| peerIP   | The IP address of this peer. | Valid IPv4 or IPv6 address.  | string | |
 | asNumber | The AS Number of this peer. | A valid AS Number, may be specified in dotted notation. | integer/string |
 
 ### Peer Scopes
 
-BGP Peers can exist at either a global scope or a node scope.  A peer's scope 
+BGP Peers can exist at either a global scope or a node scope. A peer's scope 
 determines which Calico nodes will attempt to establish a BGP session with that peer.
 
 #### Global Peer
 
-If this is a `global` scoped BGP peer, all nodes in the cluster will attempt to 
-establish a BGP connection with it.
+If this is a global scoped BGP peer, i.e. when `node` field is not specified, all nodes in
+the cluster will attempt to establish a BGP connection with it.
 
 #### Node Peer
 
-A BGP peer can also be added at the `node` scope, meaning only a single specified 
-node will peer with it. BGP peer resources of this nature must specify a `node` 
-to inform Calico which node this peer is targeting.
+A BGP peer can also be added at the node scope, i.e. when the `node` field is specified, 
```

Comment:
I think the clause that you have added here is already included in the second sentence. So.....I think the paragraph was better before.

Reply:
So this is not so much of a reorganizing the wording, we use to have a `scope` field so that's what the previous explanation was referring to, and we don';t have that anymore. You're right, the sentence is not super clear, lmk if you have a suggestion. I can explain you the concept offline

Reply:
OK, thanks for explaining. So then, I would say 
"scope, i.e. when the node field is specified, meaning only a single specified" --> 
"scope. When the `node` field is included, only the specified"